src/scripts/lib/db.js

```diff
@@ -72,8 +72,8 @@ export default class Db {
   }
 
   async getOriginFileName (domain) {
-    if (process.env.DEBUG && domain.endsWith('toggl.space')) {
-      domain = 'toggl.com';
+    if (process.env.DEBUG && domain.endsWith('track.toggl.space')) {
+      domain = 'track.toggl.com';
```

Comment:
This should remain `toggl.com` or preferably  we change the `origins.js` file to match the domain.
Currently it's set to `toggl.com` instead of `track.toggl.com`

![image](https://user-images.githubusercontent.com/1716853/94451522-21fb3700-01cc-11eb-960a-a0b9a5b09c0e.png)